doc/api/test.md

```diff
@@ -591,6 +591,9 @@ module.exports = async function * customReporter(source) {
 };
 ```
 
+The value provided for `--test-reporter` is parsed via the same rules as the
```

Comment:
This sentence doesn't really convey anything to me. What is supposed to be the takeaway? The `--import` docs don't really say a ton:

```
Preload the specified module at startup.

Follows ECMAScript module resolution rules. Use --require to load a CommonJS module.
Modules preloaded with --require will run before modules preloaded with --import.
```

I'm left wondering if I should I be looking at `--require` docs too?

Reply:
I’m struggling with good wording for this. We could also just leave it out if you want.

What I’m trying to convey is that whatever you pass to `--test-reporter` needs to follow the same rules as what you would pass to `--import`: so file URLs but not paths. In *nix systems this is rarely noticed, so it’s mostly a Windows thing where something like `--require C:\whatever` works (I think) but `--import C:\whatever` doesn’t; it needs to be `--import file:///c/whatever` or something like that. Most people pass relative paths like `--import ./something.js` or `--import some-subfolder-in-node_modules` and never give this any thought. The other thing this enables (though I don’t know why you’d want to do it) is passing a data URI or an HTTPS URL (with `--experimental-network-imports`).

Reply:
Would it be accurate to say that absolute paths to reporters must be specified as URLs, but relative paths or items from `node_modules` do not? If so, I'd be ok with saying that because I could foresee that being a gotcha to someone.

Other than my confusion around the docs, the PR LGTM.

Reply:
> Would it be accurate to say that absolute paths to reporters must be specified as URLs, but relative paths or items from node_modules do not?

Technically, no, unfortunately. A Windows relative path can have backslashes (I think) but an URL relative path needs forward slashes.

Another way of phrasing it could be that the value is anything that can be supplied to `import()` in JavaScript code. Or we can delete the line, I'm fine with that too. Or if you have another suggestion 😀

Reply:
I'm fine with saying it must be a value that can be passed to `import()` or dropping the line. I'll leave it up to you.

Reply:
Works for me!